drafts/project-management-process.md

```diff
@@ -0,0 +1,340 @@
+
+### Overview of The Roles
+
+Each Project has a [**Project Lead**](#project-lead) who is responsible for the project's Roadmap and who tends to the Goals and Milestones.  The [**Program Manager**](#program-manager) is responsible for pulling all this information together into the Organization Roadmap.
+
+We're playing with the roles of [**Product Owner**](#product-owner) and [**Project Manager**](#project-manager) but have not found a stable definition for those roles.
+
+A [**Sprint Administrator**](#sprint-administrator) maintains the pulse of weekly updates, calls, etc. and ensures that the records of these discussions are publicly available.
+
+### Overview of The Cadence
+
+We conduct public [**Weekly Updates**](#weekly-updates) where each Project's contributors and the Project Lead check in with the Roadmap, Milestones, Status Board, and relevant Goals.  These updates provide the basis for Weekly [**Sprints**](#sprints).
+
+On a quarterly basis (roughly every 3 months) we conduct [**Roadmap Updates**](#roadmap-updates) where we review the past quarter's Milestones and establish a schedule for the Milestones that will receive attention in the coming months.
+
+## Methodology: Agile? Scrum? Kanban?
+
+Agile methodology has evolved into a whole field of options -- Extreme Programming, Scrum, Kanban, etc. and there is a wealth of reference material on each approach. We like characteristics of each of these approaches, especially Scrum and Kanban, but we haven't formally adopted any one approach.  This is partially because we're a distributed team with many open source contributors spread across the world. Kanban and Scrum were initially designed for teams that are all in the same physical location, same timezone, and working for one employer.  That makes it hard (but not impossible) to apply these processes directly to our work.
+
+For now we just use a loose sense of Agile methodology. The "agile" habits we adhere to include:
+
+* Embrace change by arranging work into sprints
+* Track work as Goals on a Status Board
+* Rely heavily on Test Driven Development
+* Run Continuous Integration servers
+
+Over time, we will experiment with incorporating more elements of Kanban, Scrum and/or other approaches.
+
+## Artifacts
+
+### 📚 Projects
+
+_TODO: WTF is a Project? is a Project defined by its audience?_
+
+A **Project** is...
+
+_We **do not** use a one-to-one mapping between Projects and code bases._ Projects frequently include multiple code bases stored in multiple git repositories.
```

Comment:
Good to point out one project can involve many repos. but I think at least there will always be one repo that is the "root" of that project (eg `ipfs/go-ipfs` and `ipfs/js-ipfs` despite the many subrepos or unnamed subprojects).